src/test/java/com/notably/commons/core/Path/AbsolutePathTest.java

```diff
@@ -0,0 +1,38 @@
+package com.notably.commons.core.path;
+
+import static com.notably.testutil.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import com.notably.commons.core.path.exceptions.InvalidPathException;
+
+class AbsolutePathTest {
+
+    @Test
+    public void createAbsolutePath() {
+        assertThrows(InvalidPathException.class, () -> AbsolutePath.fromString("CS2103/notes"));
+        try {
+            assertEquals(new AbsolutePath("/CS2103/notes"), AbsolutePath.fromString("/CS2103/notes"));
+        } catch (InvalidPathException ex) {
+            System.out.println("Error");
+        }
+
+    }
+
+
+    @Test
+    public void convertAbsolutePath() {
```

Comment:
Follow the test case method's naming convention [here](https://nus-cs2103-ay1920s2.github.io/website/coding-standards/java/intermediate.html)!